app/screens/Settings/SettingsScreen.tsx

```diff
@@ -32,6 +38,62 @@ const SettingsScreen = (props) => {
       onPress: navigator.openNotifications,
       os: "ios",
     },
+    {
+      title: "Import",
+      onPress: async () => {
+        const file = await DocumentPicker.getDocumentAsync({});
+        if (file.type !== "cancel") {
+          try {
+            const data = JSON.parse(
+              await FileSystem.readAsStringAsync(file.uri)
+            );
+            if (data._persist) {
+              store.dispatch(
+                budget.actions.setMonthlyCarbonBudget(
+                  data.budget.monthlyCarbonBudget
+                )
+              );
+              store.dispatch(
+                emissions.actions.loadEmissionsFromImport(data.emissions)
```

Comment:
```
...
const SettingsScreen = (props) => {
  const navigator = navigate(props.navigation); 
  const dispatch = useDispatch(); // add this
...
```
and then here simply : `dispatch(emissions.actions.loadEmissionsFromImport(data));`